src/mono/wasm/runtime/es6/dotnet.es6.lib.js

```diff
@@ -5,16 +5,16 @@
 "use strict";
 
 #if USE_PTHREADS
-const usePThreads = `true`;
+const usePThreads = true;
 const isPThread = `ENVIRONMENT_IS_PTHREAD`;
 #else
-const usePThreads = `false`;
+const usePThreads = false;
 const isPThread = `false`;
```

Comment:
Why we are changing some strings to bools and some not?

Reply:
because some are evaluated directly and some are appended in template.